modules/auxiliary/gather/wp_bookingpress_category_services_sqli.rb

```diff
@@ -78,7 +81,21 @@ def get_sqli_object
         'vars_post' =>
             generate_vars_post(") UNION ALL SELECT (#{payload}),456,789,12,34,56,78,90,77 from wp_users-- -")
       })
-      res.get_json_document[0]['bookingpress_service_id']
+
+      if res && res.code == 200
+        begin
+          res.get_json_document[0]['bookingpress_service_id']
+        rescue JSON::ParserError => e
+          print_error("Failed to read JSON from SQLi attempt: #{e.class} - #{e.message}}")
+          return GET_SQLI_OBJECT_FAILED_ERROR_MSG
+        end
+      elsif res
+        print_error("Invalid response code #{res.code} for from SQLi attempt: response")
```

Comment:
Not sure what `: response` is referring to here. Was that meant to be the contents of the response interpolated? The wording here is a bit confusing.

Reply:
Also the use of `for from SQLi` looks like a typo here, and might tie into the above point. This looks like this line may have been edited from something earlier and some words were left in as a result?